tests/framework/pvc.go

```diff
@@ -240,6 +240,35 @@ func (f *Framework) VerifySparse(namespace *k8sv1.Namespace, pvc *k8sv1.Persiste
 	return info.VirtualSize >= info.ActualSize, nil
 }
 
+// VerifyFSOverhead checks whether virtual size is smaller than actual size. That means FS Overhead has been accounted for
```

Comment:
Can't virtual size >= actual size even with FS Overhead?

Reply:
Actually this assertion (and this function) is only valid in context where preallocation is requested. This needs to be made clear (in comment and in code). Good catch.